contrib/docker/Dockerfile

```diff
@@ -50,6 +50,8 @@ RUN pip install --upgrade setuptools pip \
     && pip install -r requirements.txt -r requirements-dev.txt \
     && rm -rf /root/.cache/pip
 
+RUN pip install gevent
```

Comment:
It might make more sense to handle this directly with the gunicorn installation.  Instead of a separate run command here, in the requirements.txt we could use the following and have pip install it automatically:
```
gunicorn[gevent]==19.8.0
```
This way the cache directories are cleaned up after as well, minimizing the image size.

Reply:
Actually, now that I look more closely at this, I'm pretty sure support for gevent/gunicorn is going to be deprecated in a future release, at least according to this PR: https://github.com/apache/incubator-superset/pull/4451

It might work best to incorporate the updated suggested command into the Dockerfile directly.

Reply:
I see.
I assume that in the end my workaround seems to be fine. Am I right?

Reply:
I haven't tested explicitly adding gevent to the Dockerfile, but if it worked for you I'll take your word on it.  I did just do a rebuild of the image using the tweak I mentioned, `gunicorn[gevent]`, and it seems to be working but I haven't exhaustively tested it.

Reply:
I also tested my idea and it's fine. I'm also opposed to putting this in requirements. If we want to do it I suggest adding another requirement file for Docker which is docker extra requirements like #7744.